tests/migration/migration.go

```diff
@@ -456,10 +456,11 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 
 			It("[test_id:6970]should migrate vmi with cdroms on various bus types", func() {
 				vmi := libvmi.NewAlpineWithTestTooling(
-					libvmi.WithMasqueradeNetworking()...,
+					append(libvmi.WithMasqueradeNetworking(),
+						libvmi.WithEphemeralCDRom("cdrom-0", v1.DiskBusSATA, cd.ContainerDiskFor(cd.ContainerDiskAlpine)),
+						libvmi.WithEphemeralCDRom("cdrom-1", v1.DiskBusSCSI, cd.ContainerDiskFor(cd.ContainerDiskAlpine)),
+					)...,
```

Comment:
If you move the `libvmi.WithMasqueradeNetworking()...` at the end, will it work?

```go
vmi := libvmi.NewAlpineWithTestTooling(
    libvmi.WithEphemeralCDRom("cdrom-0", v1.DiskBusSATA, cd.ContainerDiskFor(cd.ContainerDiskAlpine)),
    libvmi.WithEphemeralCDRom("cdrom-1", v1.DiskBusSCSI, cd.ContainerDiskFor(cd.ContainerDiskAlpine)),
    libvmi.WithMasqueradeNetworking()...,
)
```

Reply:
Self answering... this does not work with Go. :/